### ISSUE TITLE
GPL-778 As a Sulston lab user I would like to be able to print tag plate labels to Squix printers

### ISSUE THREAD

**Issue by stevieing** [sanger/gatekeeper#102]
**User story**
As a Sulston lab user I would like to be able to print tag plate labels to Squix printers. Currently we are having to go over to Oglivie to pick up labels

**Who are the primary contacts for this story**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
This could be packaged up as part of the general move over to Squix

**Comment by harrietc52** [comment]
DUP of https://github.com/sanger/gatekeeper/issues/104

**Comment by harrietc52** [comment]
Closing, duplicate of https://github.com/sanger/gatekeeper/issues/104
